src/store/Camomile.java

```diff
@@ -0,0 +1,13 @@
+package store;
+
+public class Camomile extends Flower {
+    public Camomile() {
+        super(70);
```

Comment:
for this case it is ok, better to pass value as constructor param instead of using "magic numbers"